recipes-core/everest/everest-admin-panel_0.2.bb

```diff
@@ -50,18 +50,17 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
                     file://node_modules/vuetify/LICENSE.md;md5=61626e12e39ac039026fd688eb9b1b56 \
                     file://node_modules/vuex/LICENSE;md5=761fb17e53b4b87a0507c26d4bd5bfa4"
 
-SRC_URI = "https://github.com/EVerest/everest-admin-panel/releases/download/v0.1.0/everest-admin-panel.tar.gz \
+# TODO: update shrinkwrap file
```

Comment:
Can't this be done in this PR?

Reply:
yes, I've just updated the shrinkwrap and the license listing